pkg/account/deployer/client.go

```diff
@@ -129,10 +130,10 @@ func (d *accountManagementClient) upsertPolicy(ctx context.Context, policyLevel
 	}
 
 	if len(existingPolicies) > 1 { // shouldn't happen
-		log.Warn("Found multiple policies with name %q. Updating policy with UUID %q", policy.Name, existingPolicies[0].GetUuid())
+		logr.FromContextOrDiscard(ctx).V(-1).Info("Found multiple policies with name " + policy.Name + ". Updating policy with UUID " + existingPolicies[0].GetUuid())
```

Comment:
as discussed directly: let's followup on this not actually logging as Warning but as Info, due to `logr` seemingly not working as `zapr` expects